Reference/Searching/Examine/Indexing/index.md

```diff
@@ -22,212 +13,123 @@ You can modify the built in indexes in several ways:
 
-* To customize the default analyzer used in the built in indexes
-* Change the `IValueSetValidator`
-* Modify field definitions
-* Change the location of where the indexes are stored, or
-* Completely replace the indexes with your own entirely custom implementation
+There is some documentation about this in the [Examine documentation](https://shazwazza.github.io/Examine/configuration).
 
-As an example, to change the `IValueSetValidator` for the MemberIndex in the above example:
+The easiest way to modify how a field is configured is using the ConfigureNamedOptions pattern like so:
 
 ```c#
 using Examine;
-using Umbraco.Core;
-using Umbraco.Core.Composing;
-using Umbraco.Core.Logging;
-using Umbraco.Core.Services;
-using Umbraco.Examine;
-using Umbraco.Web.Search;
-
-public class CustomizeIndexComposer : IUserComposer
-{
-    public void Compose(Composition composition)
-    {
-        // replace the default IUmbracoIndexesCreator definition
-        composition.RegisterUnique<IUmbracoIndexesCreator, CustomUmbracoIndexesCreator>();
-    }
-}
+using Examine.Lucene;
+using Microsoft.Extensions.Options;
+using Umbraco.Cms.Core;
 
-// override the default creator
-public class CustomUmbracoIndexesCreator : UmbracoIndexesCreator
+namespace MySite
 {
-    public CustomUmbracoIndexesCreator(IProfilingLogger profilingLogger, ILocalizationService languageService, IPublicAccessService publicAccessService, IMemberService memberService, IUmbracoIndexConfig umbracoIndexConfig)
-        : base(profilingLogger, languageService, publicAccessService, memberService, umbracoIndexConfig)
-    {
-    }
-
-    // note - in Umbraco 8.4 this method is obsoleted and it's advised to replace the
-    // IUmbracoIndexConfig as in the above example.
-    public override IValueSetValidator GetMemberValueSetValidator()
+    public class ConfigureExamineOptions : IConfigureNamedOptions<LuceneDirectoryIndexOptions>
     {
-        // all built in member properties to exclude (i.e. Password question, etc...)
-        var excludeFields = Constants.Conventions.Member.GetStandardPropertyTypeStubs().Keys;
+        public void Configure(string name, LuceneDirectoryIndexOptions options)
+        {
+            if (name.Equals(Constants.UmbracoIndexes.ExternalIndexName))
+            {
+                options.FieldDefinitions.AddOrUpdate(new FieldDefinition("price", FieldDefinitionTypes.Double));
+            }
+        }
 
-        // include everything except the above
-        return new MemberValueSetValidator(null, null, null, excludeFields);
+        // Part of the interface, but does not need to be implemented for this.
+        public void Configure(LuceneDirectoryIndexOptions options)
+        {
+            throw new System.NotImplementedException();
+        }
     }
 }
 ```
+This will either add the `price` field to the index, or if the field already exists, update date its type to double.
```

Comment:
```suggestion
This will ensure that the `price` field in the index is treated as a `double` type (if the `price` field does not exist in the index, it is added).
```